superset-frontend/src/dashboard/components/gridComponents/Tabs.jsx

```diff
@@ -108,6 +110,38 @@ const StyledTabsContainer = styled.div`
   }
 `;
 
+const StyledCancelXIcon = styled(Icons.CancelX)`
+  color: ${({ theme }) => theme.colors.grayscale.base};
+`;
+
+const RightDropIndicator = styled.div`
+  border: 2px solid ${({ theme }) => theme.colors.primary.base};
+  width: 5px;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  right: -4px;
+  border-radius: 2px;
+`;
+const LeftDropIndicator = styled.div`
```

Comment:
The two Indicator components appear to be identical. Do we need two identical components?

If so, how about doing it this way?
```suggestion
const LeftDropIndicator = RightDropIndactor;
```

Reply:
Much dryer, thank you for pointing that out and also supplying the emotion conducive code. I only changed your suggestion because the file is not yet in typescript.